src/app/zones/views/ZoneEgressListView.vue

```diff
@@ -144,25 +153,32 @@ function transformToTableData(zoneEgressOverviews: ZoneEgressOverview[]): ZoneEg
     const status = getItemStatusFromInsight(entity.zoneEgressInsight ?? {})
 
     return {
-      entity,
+      id,
```

Comment:
What is `id` for?

Reply:
I want to start providing IDs for all tables so that we have an easier time later on implementing mechanisms relying on the ability for identifying rows again. If every table already has an `id` (which we can ensure unlike using `name` which may not be unique within a table), adding mechanisms like a `row:select` event are easier.

Reply:
I’ve removed id.

Reply:
We do: service list view lists both services and external services and I’m pretty sure uniqueness between both resource types isn’t guaranteed.

Also, for resources that do have their own IDs, their names aren’t necessarily unique either.

Reply:
OK so we just checked with backend and it sounds like most things are unique within a mesh. There are potentially outliers where it "technically" could be an issue, in which case instead of having a unique ID we should be able to qualify it with something else:

In the case of:

1. Service/ExternalService we could use the `serviceType`
2. Anything else generally the name of the mesh.

Important to note we mostly view things by mesh anyway.

It doesn't sounds like this is anything pressing, and from the sounds of it we may have something added in the backend to gives us something unique. So I would say this is up in the air for the moment at least.

Lastly, we don't use unique ids anywhere for the moment, and for the purpose of this PR the `id` var wasn't being used at all anyway, and I don't think that previously it was being used to provide some sort of unique or qualified id anyway.

Just to be clear, no further action required from me here, just noting for historical purposes.